fspiop/v2_0_ISO20022/openapi3/components/schemas/hexBinary.yaml

```diff
@@ -1,4 +1,3 @@
 title: hexBinary
 type: string
-pattern: ^(0x|0X)?[a-fA-F0-9]+$
```

Comment:
Maybe we should align with the FSPIOP ILP packet regex.
I.e. ^[A-Za-z0-9-_]+[=]{0,2}$

Not sure why _ and == are allowed, but I am sure there it a good reason.